.travis.yml

```diff
@@ -20,22 +20,23 @@ before_install:
 
 # command to install dependencies
 install:
-  - pip install -r requirements.txt
-  - pip install -r dev-requirements.txt
+  - curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+  - source $HOME/.poetry/env
+  - poetry install -vv
 
 before_script:
-  - pip freeze
-  - python bin/setup_test_database.py
+  - poetry run python bin/setup_test_database.py
   - mkdir -p tests/resources/keys; cd tests/resources/keys; openssl genrsa -out test_private_key.pem 2048; openssl rsa -in test_private_key.pem -pubout -out test_public_key.pem; cd -
 
 # command to run tests
 script:
-  - py.test -vv --cov=peregrine --cov-report xml tests
+  - poetry run pytest -vv --cov=peregrine --cov-report xml tests
```

Comment:
I think most of our installs where we use Poetry are at 3.7 via `pyproject.toml`, but in Travis we have 3.6. But that Travis specification is moot since 1) it doesn't match poetry and 2) in Travis we'd use 3.7 anyway via poetry's python.

Reply:
Bumping travis to 3.7 anyway. Resolving this confo